<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>url.ico</normaloff>url.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 62, 80);</string>
  </property>
  <widget class="QPlainTextEdit" name="longUrlBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 152, 219);
color: rgb(236, 240, 241);</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 152, 219);
color: rgb(236, 240, 241);</string>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="shortUrlBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 62, 80);
color: rgb(236, 240, 241);
font: 22pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>URL Shortener</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
